src/WebJobs.Script.WebHost/Management/LinuxSpecialization/PackageCopyHandler.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.Azure.WebJobs.Script.Diagnostics;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Management.LinuxSpecialization
+{
+    public class PackageCopyHandler : IPackageCopyHandler
```

Comment:
>PackageCopyHandler [](http://example.com/codeflow?start=17&length=18)

also please check if we can merge with PackageDownloadHandler

Reply:
As recommended, I have merge it.